sse2neon.h

```diff
@@ -8339,6 +8420,8 @@ FORCE_INLINE uint64_t _mm_crc32_u64(uint64_t crc, uint64_t v)
     __asm__ __volatile__("crc32cx %w[c], %w[c], %x[v]\n\t"
                          : [c] "+r"(crc)
                          : [v] "r"(v));
+#elif defined(_M_ARM64)
+    crc = __crc32cd(crc, v);
```

Comment:
The first parameter should be cast to uint32_t to avoid the following warning:
`warning C4244: 'argument': conversion from 'uint64_t' to 'unsigned int', possible loss of data`